gatsby_fp/src/components/Layout/Body/Header.js

```diff
@@ -3,15 +3,14 @@ import React from "react"
 const Header = () => (
   <header id="header" className="header" role="banner">
     <div className="header_container">
-      <div className="header_site_link">
```

Comment:
Why is the className being removed? For this to conform to the cobranding template, we need to keep that.

Reply:
Hm, this is weird because when I keep the className there, the heading is rendered as the same as the picture I attached in this Jira issue.  However, when I remove it, the heading location doesn't change and appears as it does in master @mf-stevens

Reply:
I suspect this problem disappears after the other changes are made.